core/channel.go

```diff
@@ -163,15 +173,26 @@ func createChannelStep(src, dst *ProvableChain, ordering chantypes.Order) (*Rela
 	return out, nil
 }
 
-func logChannelStates(src, dst Chain, srcChan, dstChan *chantypes.QueryChannelResponse) {
-	log.Printf("- [%s]@{%d}chan(%s)-{%s} : [%s]@{%d}chan(%s)-{%s}",
-		src.ChainID(),
-		mustGetHeight(srcChan.ProofHeight),
-		src.Path().ChannelID,
-		srcChan.Channel.State,
-		dst.ChainID(),
-		mustGetHeight(dstChan.ProofHeight),
-		dst.Path().ChannelID,
-		dstChan.Channel.State,
+func logChannelStates(src, dst *ProvableChain, srcChan, dstChan *chantypes.QueryChannelResponse) {
+	relayLogger := logger.GetLogger()
+	// sLogger :=
+	GetChannelLoggerFromProvaleChain(relayLogger, src, dst).Info(
+		"channel states",
+		"msg",
+		fmt.Sprintf(
+			"src channel height: [%d] state: %s | dst channel height: [%d] state: %s",
+			mustGetHeight(srcChan.ProofHeight),
+			srcChan.Channel.State,
+			mustGetHeight(dstChan.ProofHeight),
+			dstChan.Channel.State,
+		),
+	)
+}
+
+func GetChannelLoggerFromProvaleChain(relayLogger *logger.RelayLogger, src, dst *ProvableChain) *logger.RelayLogger {
+	return logger.GetChannelLogger(
+		relayLogger,
+		src.ChainID(), src.Path().ChannelID, src.Path().PortID,
+		dst.ChainID(), dst.Path().ChannelID, dst.Path().PortID,
 	)
```

Comment:
You should also plug in the module name like below.
```
logger := logger.GetChannelLogger(...)
return logger.GetModuleLogger(logger, "core.channel")
```

Plus, you should rename `logger` package to `log` or something.

Reply:
The following style is better, maybe.
```
logger := log.GetLogger().
	WithChannel(srcChainId, srcChanId, srcPortId, ...).
	WithModule("core.channel")
```

Reply:
Add WithChannle, WIthConnection, WithModule
https://github.com/hyperledger-labs/yui-relayer/pull/92/commits/c1e80cf3f004a723722af8e614d3a840db2f6657#diff-eaacd7ed155c829fa71e358d09752203e209894410e320e6a718ff66e21ee6dcR64